### PR TITLE
feat: added default dashboard fixtures and patch

### PR DESCRIPTION
**Pull request by scmmishra**
Create default account dashboards when installing ERPNext.

**Preview:**
Screenshot_2019-07-22 Accounts Dashboard

### CODE REVIEW

File: erpnext/setup/setup_wizard/data/dashboard_charts.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+				"chart_name": "Bank Balance",
+				"timespan": "Last Year",
+				"color": "#ffb868",
+				"value_based_on": "",
+				"filters_json": json.dumps({"company": company.name, "account": bank_account}),
+				"source": "Account Balance Timeline",
+				"chart_type": "Custom",
+				"timeseries": 1,
+				"based_on": "",
+				"owner": "Administrator",
```
Comment (adityahase) [inline erpnext/setup/setup_wizard/data/dashboard_charts.py]: Can we get rid of this?

**Review by adityahase — CHANGES_REQUESTED** [review]

File: erpnext/setup/setup_wizard/operations/install_fixtures.py [hunk 2]
```diff
@@ -475,7 +475,6 @@ def install_defaults(args=None):
 
 				frappe.db.set_value("Company", args.company_name, "default_bank_account", bank_account.name, update_modified=False)
 
-				return doc
```
Comment (adityahase) [inline erpnext/setup/setup_wizard/operations/install_fixtures.py]: Were you high?

**Review by adityahase — APPROVED** [review]

File: erpnext/setup/setup_wizard/operations/install_fixtures.py (hunk 2 above)
Comment (scmmishra) [reply erpnext/setup/setup_wizard/operations/install_fixtures.py]: Someone else probably was.

File: erpnext/setup/setup_wizard/data/dashboard_charts.py (hunk 1 above)
Comment (scmmishra) [reply erpnext/setup/setup_wizard/data/dashboard_charts.py]: The empty fields? Removed them!

**Review by adityahase — APPROVED** [review]

**Review by nabinhait — APPROVED** [review]

**Pull request merged**